criteria/expression_child.go

```diff
@@ -0,0 +1,20 @@
+package criteria
+
+// ChildExpression represents the negation operator
```

Comment:
This comment doesn't make sense.

Reply:
Fixed in 28909d05bb23f2acfc1046f60e7bed5fe3eac292